### PR TITLE
Intro lesson1

### PR DESCRIPTION
**Pull request by yurapukish**
Add HomeWork1, requirements .txt file, install flake8

### CODE REVIEW

**Comment by antohaUa** [comment]
Here is full linter log 
(venv) [antoha@dlhn test_project]$ ./bin/lint ./W-HOMETASKS/L01/l01_h01_PukishYurii.py
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:1:1: D100 Missing docstring in public module
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:9:1: N400: Found backslash that is used for line breaking
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:13:1: N400: Found backslash that is used for line breaking
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:13:16: P101 format string does contain unindexed parameters
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:15:1: S101 Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:19:1: N400: Found backslash that is used for line breaking
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:21:1: S101 Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
./W-HOMETASKS/L01/l01_h01_PukishYurii.py:21:48: W292 no newline at end of file

File: Lesson1/HomeWork1.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# we have norway text in old style formatting
```
Comment (antohaUa) [inline Lesson1/HomeWork1.py]: Such comments better organize as module docstring aka multiline """ Some description ... """

File: Lesson1/HomeWork1.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# re-write the same text as:
+# #1 string with format() call
+# #2 f-string
+# use linter(https://github.com/wemake-services/wemake-python-styleguide)
+# to check your new created python module for possible linter errors
+# try to run code from pycharm/command line
+
+norway_text = 'Automatisering akselererer %syeblikket da roboter vil ' \
```
Comment (antohaUa) [inline Lesson1/HomeWork1.py]: Not critical but I recommend to use smth aka 
```
text = ('abcd'
        'fegk')
```
Such construction without slashes more readable + will solve linter N400 warnings

File: Lesson1/HomeWork1.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+              'erobre planeten v%sr. (%s)' % ('ø', 'å', 'Æ')
+
+# 1 Using format() method:
+norway_text2 = 'Automatisering akselererer {}yeblikket da roboter vil ' \
```
Comment (antohaUa) [inline Lesson1/HomeWork1.py]: Here better to use indexes  aka {0} {1} {2}

File: Lesson1/HomeWork1.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+               'erobre planeten v{}r. ({})'.format('ø', 'å', 'Æ')
+assert norway_text == norway_text2, 'Not equal'
+
+# 2 Using f-string:
+character1, character2, character3 = 'ø', 'å', 'Æ'
+norway_text3 = f'Automatisering akselererer {character1}yeblikket ' \
```
Comment (antohaUa) [inline Lesson1/HomeWork1.py]: Just note that without variables text could be placed directly.  aka {'ø'}.
From the first look it has no sense but could be as sign for understanding where in future to place variables.

File: Lesson1/HomeWork1.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+               f'da roboter vil erobre planeten v{character2}r. ({character3})'
+assert norway_text == norway_text3, 'Not equal'
```
Comment (antohaUa) [inline Lesson1/HomeWork1.py:25]: Assert accepted But don't forget about printing too. Automation logs will be more readable. And if error occurs much easier to understand where exactly we fail.

**Review by antohaUa — CHANGES_REQUESTED** [review]

**Comment by yurapukish** [comment]
Close current pull request, cause code need some improvements

**Pull request closed without merging**